core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -541,40 +492,18 @@ private DefaultFunctionResolver datediff() {
             nullMissingHandlingWithProperties(DateTimeFunction::exprDateDiff),
             LONG,
             TIME,
-            TIMESTAMP),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprDateDiff),
-            LONG,
-            TIMESTAMP,
-            DATETIME),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprDateDiff),
-            LONG,
-            DATETIME,
-            TIMESTAMP),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprDateDiff),
-            LONG,
-            TIME,
-            DATETIME),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprDateDiff),
-            LONG,
-            DATETIME,
-            TIME));
+            TIMESTAMP));
   }
 
   /**
-   * Specify a datetime with time zone field and a time zone to convert to.<br>
-   * Returns a local date time.<br>
-   * (STRING, STRING) -> DATETIME<br>
-   * (STRING) -> DATETIME
+   * Specify a datetime with time zone field and a time zone to convert to. Returns a local date
+   * time. (STRING, STRING) -> TIMESTAMP (STRING) -> TIMESTAMP
```

Comment:
I believe this might be my mistake from a rebasing conflict blunder, fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/f6edc6be62abbe3a3974665e8d51c515584234e0